color-theme-sanityinc-tomorrow.el

```diff
@@ -51,6 +53,9 @@
 (require 'color)
 
 (eval-when-compile (require 'ansi-color))
+(unless (fboundp 'declare-function)
```

Comment:
Rather than define this to nil, why not just only say `(when (fboundp 'declare-function) (declare-function ...))`?

Reply:
works for me.